### ISSUE TITLE
[a11y support] PDF Audit - VA Claims - Supporting Forms: Form 21P-0847

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#78739]
> [!NOTE]
>  An audit is to be performed on the following **VA Claims - Supporting Form:**
>  [Form 21P-0847](https://www.va.gov/supporting-forms-for-claims/#request-to-be-a-substitute-cla) Request to be a substitute claimant for a deceased claimant

# PDF Audit Process
This process is thorough and designed to check that all aspects of document accessibility and confirm compliance with a matching accessibility report.

> [!TIP]
> CAIA is currently drafting documentation relating to: [How to create an accessible PDF](https://github.com/department-of-veterans-affairs/va.gov-team/blob/710debbef61acb4caeba60a6886218c5a2a8be81/teams/CAIA/accessibility/create-accessible-PDF.md#test-your-pdf) Review this GitHub page, to learn more about PDF auditing, remediation and tools.

## Testing Tool

To ensure that the PDF is accessible, it will need to be tested with a special tool that tests the PDF document for various accessibility standards, such as Section 508 and WCAG 2.0 AA.

### Preferred CAIA Testing Tool
**CommonLook PDF Validator**, also known as **Allyant**, is the CAIA team's testing tool of choice. It's a free plugin for Adobe Acrobat Pro or DC that tests and verifies PDF documents.

### Tool Used in this PDF Audit
- [ ] CommonLook PDF Validator (Allyant)
- [x] Adobe Acrobat DC
- [ ] axesPDF
- [ ] AccessibilityNow® Remediate
- [x] Manual testing
- [ ] Other

## PDF Audit Areas
- [x] Tag Tree
- [ ] Artifact Images
- [x] Verify alt text in images 
- [x] Heading styles
- [x] Verify reading order
- [x] Verify correct tag usage
- [ ] Look for color issues
- [x] Review overall tag usage
- [x] Review table tags
- [x] Review form tags
- [x] Review link tags
- [x] Review for descriptive links
- [ ] Review artifact tags
- [x] Review header/footer tags

## PDF Remediation Needed
- [ ] Set the tabbing order to follow the document structure
- [ ] Add the document tag
- [ ] Add bookmarks
- [ ] Clean up empty tags
- [ ] Add Metadata Information (adding the checkbox for 508, WCAG, PDF/UA or HHS)
- [x] #81942
- [x] #81943
- [ ] Fix tagging errors in tables
- [ ] Fix tagging errors in forms
- [ ] Fix tagging errors in links
- [ ] Fix tagging errors in artifact tags

## PDF Accessibility Reporting
- [ ] Run CommonLook Verification on structural compliance
- [ ] Run CommonLook Verification on accessibility compliance
- [ ] Generate and save the CommonLook Accessibility Report according to the chosen standard (WCAG2.1AA, PDF/UA/HHS/508)

**Comment by sara-amanda** [comment]
## Ticket Status Update
- Moving this ticket to `Validate` based on conversation today with @EvanAtCoforma. 
- [This ticket will ultimately be dependent on the following ticket comment.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/67994#issuecomment-2101528393)

cc: @SarahKay8 @coforma-terry

**Comment by sara-amanda** [comment]
## PDF Audit Next Steps 
> [!IMPORTANT] 
> - **Audit Status**: Completed and provided to OCTO.
> - **Tickets:** Only the items that are checked with tickets are the remediation needs.
> - **Recommendation:** "Unfortunately, these forms are currently inaccessible[ per our testing](https://dsva.slack.com/archives/C072R8PSAKU/p1715268747006459), so we believe it is imperative that we or another PDF specialist get access to these forms to fix them."
> - **PDF Remediation Status:** Work not requested at this time.

### Ticket Update
Closing this ticket, per Martha via [Slack on June 3, 2024](https://dsva.slack.com/archives/D04NHP2FGEP/p1717433725780809?thread_ts=1717428202.653759&cid=D04NHP2FGEP).

- VA is working through bugs with Adobe.
- Our offer to remediate these forms is not needed at this time. 
- PDF audit efforts to be entered in CAIA Closed Ticket Tracking - verified with @EvanAtCoforma  (@NaomiPMC and @strelichl )
